pkg/cloudprovider/vsphere/nodemanager.go

```diff
@@ -138,6 +149,25 @@ func (nm *NodeManager) shakeOutNodeIDLookup(ctx context.Context, nodeID string,
 	return nil, err
 }
 
+func returnIPsFromSpecificFamily(family string, ips []string) []string {
+	var matching []string
+
+	for _, ip := range ips {
+		if ErrOnLocalOnlyIPAddr(ip) != nil {
+			klog.V(4).Info("IP is Local Only")
+			continue
+		}
+
+		if strings.EqualFold(family, IPv6Family) && net.ParseIP(ip).To4() == nil {
```

Comment:
Hi @dvonthenen,

Should `To4` be `To6`?

Reply:
Hi @dvonthenen,

Never mind. I see what you're doing here now. I missed it the first time.

Reply:
No worries!